src/ru/skypro/Employee.java

```diff
@@ -0,0 +1,66 @@
+package ru.skypro;
+
+import java.util.Objects;
+
+public class Employee {
+    private static int count;
+    private int id;
```

Comment:
стоит отделять статические поля от других для их группировки
```suggestion
    private static int count;
    
    private int id;
```